packages/flame_behaviors/test/src/behaviors/propagating_collision_behavior_test.dart

```diff
@@ -59,7 +63,7 @@ class _CollisionBehaviorAtoComponent
 void main() {
   final flameTester = FlameTester(TestGame.new);
 
-  group('CollisionBehavior', () {
+  group('$CollisionBehavior', () {
```

Comment:
If the class name ever changes, either it will throw an error because you didnt "refactor" or it will automatically rename here as well as it just points to a reference

Reply:
Holy crap

Reply:
Credits to @alestiago, saw him do it a few months ago